production-elasticsearch/src/test/java/io/jaegertracing/kubernetes/ElasticearchETest.java

```diff
@@ -17,24 +17,28 @@
 import io.jaegertracing.kubernetes.deployment.BaseETest;
 import java.io.IOException;
 import java.util.UUID;
+import java.util.concurrent.TimeUnit;
 import okhttp3.Request;
 import okhttp3.Response;
 import org.junit.Before;
+import org.junit.Ignore;
 
 import static org.awaitility.Awaitility.await;
 
 /**
  * @author Pavol Loffay
  */
-public class ProductionETest extends BaseETest {
+public class ElasticearchETest extends BaseETest {
 
   /**
    * We need to initialize ES storage, before we proceed to tests for two reasons:
    * 1. sometimes first span is not stored
    * 2. jaeger-query returns 500 is ES storage is empty (without indices) https://github.com/jaegertracing/jaeger/issues/464
    */
   @Before
-  public void before() {
+  public void before() throws InterruptedException {
+    TimeUnit.SECONDS.sleep(15);
```

Comment:
Nope :) It's better to ping for something from time to time than just sleeping for a long time and hoping it will be enough. Besides, waiting 15 seconds won't scale: if you have 10 tests, that's already two and a half minutes of just waiting...

As the only test on this class is being ignored, can't you just remove this `@Before` entirely? Or even the whole test?

Reply:
Wow I forgot to remove this sleep statement. 

> As the only test on this class is being ignored, can't you just remove this @Before entirely? Or even the whole test? 

The class extends different test class. Before is still needed.